<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="8" column="1">
       <widget class="QPushButton" name="pushButton_OpenBlu">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QTextEdit" name="textBrowser">
        <property name="maximumSize">
         <size>
          <width>16777210</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QTextEdit" name="resTexEdit"/>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="pushButton_Send">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="pushButton_CloseBlu">
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="pushButton_scanblt">
        <property name="text">
         <string>扫描</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QSlider" name="verticalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
